golang/pkg/cli/cli.go

```diff
@@ -9,6 +9,23 @@ import (
 	"github.com/dyrector-io/dyrectorio/golang/internal/version"
 )
 
+// flags
+const (
+	FlagDisableCrux           = "disable-crux"
+	FlagDisableCruxUI         = "disable-crux-ui"
+	FlagLocalAgent            = "local-agent"
+	FlagWrite                 = "write"
+	FlagDebug                 = "debug"
+	FlagPrioritizeLocalImages = "prioritize-local-images"
+	FlagDisablePodmanChecks   = "disable-podman-checks"
+	FlagConfigPath            = "config"
+	FlagPrefix                = "prefix"
+	FlagImageTag              = "image-tag"
```

Comment:
Makefile upd uses `imagetag` instead of `image-tag` :)

Reply:
Yes, good catch, found some additional leftovers too.